sunbeam-python/sunbeam/plugins/telemetry/plugin.py

```diff
@@ -168,7 +169,15 @@ def is_skip(self, status: Optional[Status] = None) -> Result:
                 return Result(
                     ResultType.SKIPPED, "Ceilometer/Gnocchi applications missing"
                 )
-        except JujuException as e:
+
+            # Check if any one of gnocchi unit is active
+            app = run_sync(self.jhelper.get_application("gnocchi", OPENSTACK_MODEL))
+            if not any([unit.workload_status == "active" for unit in app.units]):
```

Comment:
```suggestion
            if not any(unit.workload_status == "active" for unit in app.units):
```

Collecting into a list is unnecessary

Reply:
Well, I can not block the PR for this, since in any case, there won't be thousands of gnocchi units. It just as a rule of thumb, I don't like collecting into a list when unnecessary.

Reply:
resolved